test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -584,7 +586,7 @@ def test_pytorch_telemetry_gpu(
 @pytest.mark.model("N/A")
 @pytest.mark.parametrize("ec2_instance_type", PT_EC2_CPU_INSTANCE_TYPE, indirect=True)
 def test_pytorch_telemetry_cpu(pytorch_training, ec2_connection, cpu_only, pt15_and_above_only):
-    execute_ec2_training_test(ec2_connection, pytorch_training, PT_TELEMETRY_CMD)
+    execute_ec2_training_test(ec2_connection, pytorch_training, PT_TELEMETRY_CMD, timeout=900)
```

Comment:
Curious, why the lower timeout?

Reply:
it's one of the possible culprits of the codebuild timeouts. It's flaky so couldn't exactly replicate it in the local context. Adding timeout here to see if it fails the codebuild. Also it'd be helpful to fail the timed-out tests since the rest of the test cache could still be uploaded to prevent retries.